CHANGELOG.md

```diff
@@ -8,9 +8,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 The changes not yet present in any release are listed in this section.
 
+### Changed
+
+* The acpi_call module loading is managed by systemd rather than in */etc/modules*.
+
 ### Fixed
 
 * Systemd service runs asus-fan-control on resume from hibernate.
+* The acpi_call module is loaded even after being updated.
```

Comment:
I have changed your changelog item and split it into two, each connected only with one bug/feature.

Also, as you might have noticed, I have changed `acpi_call` to acpi_call (even though I was using the former in the past). The reason is simple. I am trying to use backticks (\`) only in direct connection with the terminal. In this case, I would use `acpi_call` when talking about the `modprobe` argument, otherwise it is just a program name for me. Does it make sense? Feel free to express yourself and tell me what is your opinion. Appreciated! :+1: 

Also, as you have already noticed, I use asterisks (\*) in connection with a file path/name. Thank you so much for respecting that! :heart:

Reply:
Totally makes sense, I'm just used to use monospace for package/command/module names and paths. But I gotcha, there's no problem with that! :+1: